packages/core/test/CardanoNode/mocks.ts

```diff
@@ -1,9 +1,12 @@
 import { Cardano, EraSummary, HealthCheckResponse, StakeDistribution } from '../../src';
 
 const mockEraSummaries: EraSummary[] = [
-  { parameters: { epochLength: 21_600, slotLength: 20_000 }, start: { slot: 0, time: new Date(1_563_999_616_000) } },
   {
-    parameters: { epochLength: 432_000, slotLength: 1000 },
+    parameters: { epochLength: 21_600, slotLength: Cardano.util.Milliseconds(20_000) },
```

Comment:
I just realised the `time` util is nested in the `Cardano` scope unnecessarily. Rather than creating a new task, let's just fix this now by hoisting to the top-level `util` dir.

Reply:
Updated here: [9cb35e4](https://github.com/input-output-hk/cardano-js-sdk/pull/576/commits/9cb35e4dfc11a9777125df76e036bce5cf8bf412)